helm/choreo-connect/README.md

```diff
@@ -251,6 +251,84 @@ helm install <RELEASE_NAME> wso2/choreo-connect --version 1.0.0-1 --namespace <N
 
 You will be able to see the Analytics data when you log into Choreo Analytics Portal.
 
+### 4. Obtain the external IP
+
+Obtain the external IP (`EXTERNAL-IP`) of the Choreo Connect Ingress resources, by listing down the Kubernetes Ingresses.
+
+#### 1. Deployment Mode: Standalone
+
+```  
+kubectl get ing -n <NAMESPACE>
+```
+
+The output under the relevant column stands for the following.
+
+Choreo Connect Adapter
+
+- NAME: Metadata name of the Kubernetes Ingress resource (Default to "<RELEASE_NAME>-choreo-connect-adapter-ingress")
+- HOSTS: Hostname of the Choreo Connect Adapter (Default to "adapter.wso2.com")
+- ADDRESS: External IP (`EXTERNAL-IP`) exposing the Chore Connect Adapter to outside of the Kubernetes environment
+- PORTS: Externally exposed service ports of the Choreo Connect Adapter
+
+Choreo Connect Gateway
```

Comment:
Choreo Connect Gateway Runtime

Reply:
fixed with https://github.com/wso2/kubernetes-microgateway/pull/24/commits/81df6f4469293878efab15f4464b47f91c964c8e

Reply:
Updated this to `Choreo Connect Router` with https://github.com/wso2/kubernetes-microgateway/pull/24/commits/5b7a6744b28bfdfba62aa5369c9a4e157091408f